nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -1050,7 +1050,7 @@ services.transmission.settings.rpc-bind-address = "0.0.0.0";
    </listitem>
    <listitem>
     <para>
-      The option <option>defaultPackages</option> was added. It installs the packages <package>perl</package>, <package>rsync</package> and <package>strace</package> for now. They were added unconditionally to <option>systemPackages</option> before, but are not strictly necessary for a minimal NixOS install. You can set it to an empty list to have a more minimal system. Be aware that some functionality might still have an impure dependency on those packages, so things might break.
+      The option <option>defaultPackages</option> was added. It installs the packages <package>nano</package>, <package>perl</package>, <package>rsync</package> and <package>strace</package> for now. They were added unconditionally to <option>systemPackages</option> before, but are not strictly necessary for a minimal NixOS install. You can set it to an empty list to have a more minimal system. Be aware that some functionality might still have an impure dependency on those packages, so things might break.
```

Comment:
this has to go in the changelog of 21.03 (which might not exist yet)

Reply:
Can't we just backport this pr? Nano is still installed by default, so this shouldn't affect any setups.

Reply:
Yes, we could.

I want that defaultPackages can really be set to empty without causing any issues, so i would like more testing on unstable...

In this case it might be ok.

I have micro installed and use it most of the time, but git still opens nano. I haven't made the effort to change that yet. Have you tested for example such a case? And there might be edgecases like with rsync and nixops.

Reply:
I agree with @davidak - I'm a big fan of removing packages causing side effects but let's keep this in `unstable` to bake for the next 6 months... we'll catch most issues that way.

Reply:
Thanks for the thorough answers. I didn't think about the edge cases, so stabilizing this first makes sense, I'll update the pr.
Also, can someone proofread the release notes? English isn't my first language.